index.html

```diff
@@ -3246,6 +3246,55 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of <a>properties</a> serving as possible extension points. While
+some implementers signaled interest in these, their inclusion in this specification
+was considered to be premature; these extension points might be more
+formally defined in future versions of this specification. While these extension
+points are reserved in the base context, their usage is undefined by this
+specification and implementers are cautioned that usage of these properties is
+considered experimental.
+        </p>
+
+        <p class="issue" title="Reserved properties under debate">
+The following extension point properties are under consideration for being
+marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
+`evidence`, `render`, and `confirmationMethod`.
```

Comment:
As mentioned elsewhere, we might want to use `renderHint` both to be clear that it's just a suggestion for rendering and to follow the convention of using nouns.

```suggestion
`evidence`, `renderHint`, and `confirmationMethod`.
```

I continue to think the name `confirmationMethod` has some problems but that is being considered elsewhere as well and we could change it here later as needed.

Reply:
Why do we need to reserve these in the core spec? 

Can't we just define them in the vc spec directory?

Thats what we did with DID core, the did core context does not define things that are in the did spec registries...

https://github.com/w3c/did-spec-registries/blob/main/vocabs/v1/context.jsonld

```
{
  "@context": {
    "@protected": true,
    "id": "@id",
    "type": "@type",

    "alsoKnownAs": {
      "@id": "https://www.w3.org/ns/activitystreams#alsoKnownAs",
      "@type": "@id"
    },
    "assertionMethod": {
      "@id": "https://w3id.org/security#assertionMethod",
      "@type": "@id",
      "@container": "@set"
    },
    "authentication": {
      "@id": "https://w3id.org/security#authenticationMethod",
      "@type": "@id",
      "@container": "@set"
    },
    "capabilityDelegation": {
      "@id": "https://w3id.org/security#capabilityDelegationMethod",
      "@type": "@id",
      "@container": "@set"
    },
    "capabilityInvocation": {
      "@id": "https://w3id.org/security#capabilityInvocationMethod",
      "@type": "@id",
      "@container": "@set"
    },
    "controller": {
      "@id": "https://w3id.org/security#controller",
      "@type": "@id"
    },
    "keyAgreement": {
      "@id": "https://w3id.org/security#keyAgreementMethod",
      "@type": "@id",
      "@container": "@set"
    },
    "service": {
      "@id": "https://www.w3.org/ns/did#service",
      "@type": "@id",
      "@context": {
        "@protected": true,
        "id": "@id",
        "type": "@type",
        "serviceEndpoint": {
          "@id": "https://www.w3.org/ns/did#serviceEndpoint",
          "@type": "@id"
        }
      }
    },
    "verificationMethod": {
      "@id": "https://w3id.org/security#verificationMethod",
      "@type": "@id"
    }
  }
}
```

If it is in the vc data model, it belongs in the v2 context, if it is in the vc specs directory and not the vc-data-model, it does not belong in the vc-data-model v2 context.

Reply:
> Why do we need to reserve these in the core spec?

Because the term will be defined/reserved in the base context.

> Can't we just define them in the vc spec directory?

We can't, for the reasons above.

There is an important difference between: "The WG is reserving this property" and "Some random person on the Internet is reserving this property". I'd argue that we don't want to give random people on the Internet the ability to reserve terms in the core context (nor could they, the WG probably wouldn't allow that).

Reserving a property by the WG signals to the ecosystem that the WG believes that there is a subset of the group that believes that an extension point is reserved, but the WG didn't have time to work on it in this iteration of the specification.

Reply:
> As mentioned elsewhere, we might want to use `renderHint` both to be clear that it's just a suggestion for rendering and to follow the convention of using nouns.

-1 to `renderHint` -- they're not hints, they're complete instructions on how to render the credential that the issuer has provided. 

What about `renderer` instead? That's the noun form of `render`, and it is aligned with usage in the field of computer science (graphics model renderer, audio model renderer, 3d print renderer).

> I continue to think the name `confirmationMethod` has some problems but that is being considered elsewhere as well and we could change it here later as needed.

Agree that `confirmationMethod` is problematic and needs to be bike shedded.

Reply:
I could live with `renderer` even though that sounds more like an application or functional component whereas the values for this field would be parameters or information (or "complete instructions" as you said) describing how to render the credential.

Reply:
`howToRender`?
`renderGuidance`?
`renderMethod`?
`renderTemplate`?

Reply:
`renderMethod` sounds good and falls inline with `confidenceMethod`.

Reply:
I'm not opposed to `renderMethod`